torch/_dynamo/variables/nn_module.py

```diff
@@ -186,6 +190,45 @@ def record_nn_module_stack():
                 if is_lazy:
                     self.module_type = mod.cls_to_become
 
+                # Custom name denotes that we do not want to just call the module
+                if custom_name:
```

Comment:
This logic belongs in `call_method`.

Reply:
Agreed, then we don't need the arg here. I was going to do it as a call_function but changed my mind.